### DISCUSSION TITLE
Vermisse 2-3 Messpunkte aus evcc

### DISCUSSION THREAD

**Discussion by ben-germany** [marq24/ha-evcc#21] [Ideas]
Hallo, erstmal...top, was du da entwickelt hast. Das spart etwas Zeit die Entitäten in der config.yaml selbst zu erzeugen. Gerade mit HA macht es ja auch Sinn, das Energie-Modul von HA zu verwenden. Dazu bieten sich 2 Messpunkte an, die leider (noch) nicht als Entitäten zur Verfügung stehen. Ich würde ungerne deswegen wieder die Entitäten via MQTT/config.yaml selbst erzeugen wollen.
Es geht um "solarPercentage" und "chargedKWh", unter statistics/total (gerne zusätzlich auch 30d, 365d, thisYear) aber "total" wäre für HA am sinnvollsten, damit spart man sich Helfer anzulegen, die dann letztendlich die Summe bilden, die EVCC eigentlich schon gebildet hat.

**Comment by marq24** [comment]
da gab's Gründe warum ich (bislang) zu faul war die `statistics` auszulesen... mal sehen wann das Wetter wieder so gar keine Lust macht, nach draußen zu gehen...

**Reply by ben-germany** [reply]
Wäre top...ich denke das wird so manchen interessieren.

**Reply by marq24** [reply]
Du bist zumindest der Erste der danach fragt ;-)

**Reply by UDicke** [reply]
Ich habe auch für das Energiedashboard die "loadpoints" (Wallboxen) in HA integriert. Wenn die Energiedaten von evcc da wären, könnte man darauf verzichten.

Man muss ja auch erst mal auf die Idee kommen um danach zu fragen.😊

**Comment by marq24** [comment]
Here you go...
https://github.com/marq24/ha-evcc/releases/tag/2024.10.0

**Reply by UDicke** [reply]
👍 und das bei dem Wetter!😊 👍

**Reply by marq24** [reply]
iss' ganz schön frisch heut' Morgen ;-)

**Reply by ben-germany** [reply]
Top....gerade mal schnell installiert und funktioniert. Danke für die schnelle Reaktion. 
1 Kaffee geht auf mich....sollte angekommen sein.

**Reply by marq24** [reply]
Ja ist angekommen & schon getrunken - vielen Dank

**Reply by Beme99** [reply]
Danke für Deine Extension, zur neuen Version hätte ich eine kurze Anmerkung, es hat sich ein kleiner Schreibfehler eingeschlichen: 356 statt 365

![image](https://github.com/user-attachments/assets/ea3515a9-ca3b-4d09-aa62-74e96e443cd3)

**Comment by lucode** [comment]
Gehe ich richtig in der Annahme, dass die Integration keine neuen im API exponierten Werte selber erkennen und Aufbereiten kann. Oder ob dies technisch überhaupt möglich wäre. 
Frage weil mich die Vision umtreibt in evcc API oder Home Assistant templates zu haben, die dann von der Integration aufgenommen werden. Die neuen Entitäten könnten ev. auch manuell via die Optionen definiert werden.

**Reply by marq24** [reply]
wenn es neue Objekte/Werte in der evcc API gibt, dann müssen diese in der Integration implementiert werden...
